OCLOMRS-204: Concepts HEAD version card in an Implementation Dictionary owned by an Organisation should display live data

alexmochu opened the pull request:
# JIRA TICKET NAME:
[Concepts HEAD version card in an Implementation Dictionary owned by an Organisation should display live data](https://issues.openmrs.org/browse/OCLOMRS-204)

# Summary:
Currently, an implementation dictionary owned/managed by an organisation is not displaying the correct details in regards to the number of concepts.

Therefore it should be able to display the correct live data.

Michyjones commented:
LGTM

dkayiwa commented:
Can you make the commit message adhere to these guidelines? https://wiki.openmrs.org/display/docs/Pull+Request+Tips

alexmochu commented:
@dkayiwa Thanks for the feedback. I have resolved the commit message issue.

dkayiwa commented:
@alexmochu there are still other things that you have missed out on https://wiki.openmrs.org/display/docs/Pull+Request+Tips

alexmochu commented:
@dkayiwa I have reviewed the Pull Request wiki and also asked my teammates to review the PR again. The following are the key tips that I have implemented while raising the PR: Branch name is equals to the ticket name, I have referenced the PR with the ticket JIRA id, the PR is up to date with master, I have squashed my commits and the test/build are passing. 
Could you be more specific in what I am not doing right.
